Dockerfile

```diff
@@ -4,8 +4,9 @@ RUN apk update && \
     apk add --no-cache openssl openssh && \
```

Comment:
I know that your PR didn't touch this part but can you please merge all of these `apk add` commands into a single execution? It's probably going to speed up the build process a bit.

Reply:
sure, I also thought of more things to speed up build process like creating own images to compile the cppkafka and poco forks and removing obsolete dependencies. Will create a ticket for that and include all of these changes in a separate PR.